content/home/marketing/declarative.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Declarative
+title: Declarativa
```

Comment:
```suggestion
title: Declarativo
```

Só para aliviar minha agonia: A gente tende a tratar de React no masculino,  "O React". Mas vi que você está usando aqui no feminino, tanto aqui como em "Baseada em componentes". Nesse caso, vc está tentando concordar com "a Biblioteca React" ?

Se for, eu entendo, mas acho que fica meio esquizofrênico colocar os títulos principais no feminino e passar a usar no masculino no restante da documentação toda. Será que não seria uma boa já eleger o gênero correto logo agora pro leitor se acostumar?